.fixtures.yml

```diff
@@ -1,4 +1,4 @@
 ---
 fixtures:
   symlinks:
```

Comment:
I don't think you need a .fixtures.yml file at all.  I have other acceptance tests (e.g. simp-utils) for which there is no .fixtures.yml.